extensions/adobe/experience/campaign/profile.example.1.json

```diff
@@ -0,0 +1,442 @@
+{
+  "xdm:identities": [
+    {
+      "xdm:id": "someone@example.com",
+      "xdm:namespace": {
+        "xdm:code": "Email"
+      }
+    },
+    {
+      "xdm:id": "123-456",
+      "xdm:namespace": {
+        "xdm:code": "BB111"
+      }
+    }
+  ],
+  "xdm:person": {
+    "xdm:name": {
+      "xdm:givenName": "Jane",
+      "xdm:middleName": "F",
+      "xdm:surname": "Doe"
+    },
+    "xdm:birthMonth": 1,
+    "xdm:birthDay": 3,
+    "xdm:birthYear": 1996,
+    "xdm:gender": "female"
+  },
+  "xdm:workAddress": {
+    "@id": "https://data.adobe.io/entities/address/123",
+    "xdm:primary": false,
+    "xdm:street1": "345 Park Ave",
+    "xdm:city": "San Jose",
+    "xdm:stateProvince": "US-CA",
+    "xdm:postalCode": "95110",
+    "xdm:country": "United States",
+    "xdm:countryCode": "US",
+    "schema:latitude": 37.3382,
+    "schema:longitude": 121.8863,
+    "xdm:status": "active",
+    "xdm:lastVerifiedDate": "2018-01-02"
+  },
+  "xdm:workEmail": {
+    "xdm:primary": false,
+    "xdm:address": "jsmith@xyzinc.com",
+    "xdm:label": "John Smith",
+    "xdm:type": "work",
+    "xdm:status": "active"
+  },
+  "xdm:mobilePhone": {
+    "xdm:primary": true,
+    "xdm:number": "1-408-888-8888",
+    "xdm:status": "active"
+  },
+  "xdm:optInOut": {
+    "https://ns.adobe.com/xdm/channels/email": "pending",
+    "https://ns.adobe.com/xdm/channels/phone": "out",
+    "https://ns.adobe.com/xdm/channels/sms": "in",
+    "https://ns.adobe.com/xdm/channels/fax": "not_provided",
+    "https://ns.adobe.com/xdm/channels/direct-mail": "not_provided",
+    "https://ns.adobe.com/xdm/channels/apns": "not_provided",
+    "xdm:globalOptout": false
+  },
+  "xdm:orgUnit": {
+    "@id": "https://data.adobe.io/org-apparals-men"
+  },
+  "xdm:geoUnit": {
+    "@id": "https://data.adobe.io/geo-france"
+  },
+  "xdm:segments": [
+    {
+      "xdm:segmentID": {
+        "@id":
+          "https://data.adobe.io/entities/segmentidentity/04a81716-43d6-4e7a-a49c-f1d8b3129ba9",
+        "xdm:namespace": {
+          "xdm:code": "AAM"
+        }
+      },
+      "xdm:version": "15",
+      "xdm:timestamp": "2018-04-26T15:52:25+00:00",
+      "xdm:validUntil": "2019-04-26T15:52:25+00:00",
+      "xdm:status": "existing"
+    },
+    {
+      "xdm:segmentID": {
+        "@id":
+          "https://data.adobe.io/entities/identity/53cba6b2-a23b-454a-8069-fc41308f1c0f",
+        "xdm:namespace": {
+          "xdm:code": "AAM"
+        }
+      },
+      "xdm:version": "3",
+      "xdm:lastQualificationTime": "2018-04-26T15:52:25+00:00",
+      "xdm:validUntil": "2018-04-27T15:52:25+00:00",
+      "xdm:status": "realized"
+    }
+  ],
+  "xdm:emailScores": {
```

Comment:
I assume `xdm:emailScores` column will be inside the `xdm:journeyAI` object.

Reply:
Good catch @josemathew!  @cdegroot-adobe the example you posted missed the "journeyAI" sub-object in profile, can you fix before we merge?

Reply:
Thanks @kstreeter . @cdegroot-adobe  column should be `xdm:emailScore`, as defined in the schema definition